halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/providers/azure/AzureProvider.java

```diff
@@ -17,8 +17,14 @@
 package com.netflix.spinnaker.halyard.config.model.v1.providers.azure;
 
 import com.netflix.spinnaker.halyard.config.model.v1.node.HasImageProvider;
+import lombok.Data;
 
 public class AzureProvider extends HasImageProvider<AzureAccount, AzureBakeryDefaults> {
+//  @Data
```

Comment:
Can this section be removed instead of being commented out?

Reply:
removed